CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Habitat CHANGELOG
 
+<!-- latest_release -->
```

Comment:
I'm not sure how this will interact with your `0.56.0-dev` version. I'll need some time to perform some tests to make sure it behaves as expected (or make any changes if necessary).

Reply:
Any update on this @tduffield? I still need to address your other comments, but I'm still generally uncertain how to test this at all.

Reply:
gah! I'm sorry! This feel off my radar. I'll make a note to follow up here in the next day or so.